src/python/WMCore/WMSpec/WMWorkload.py

```diff
@@ -673,6 +673,15 @@ def removeTask(self, taskName):
         self.data.tasks.tasklist.remove(taskName)
         return
 
+    def getSiteWhitelist(self):
+        """
+        Get the site white list from our data
```

Comment:
"Get the site white list for the workflow"

Reply:
ok, done